multimodal/tests/unittests/predictor/test_predictor_model_dump.py

```diff
@@ -33,7 +33,7 @@ def test_dump_timm_image():
         time_limit=5,
         seed=42,
     )
-    predictor_1.dump_timm_image(path=model_dump_path)
+    predictor_1.dump_multimodal(path=model_dump_path)
```

Comment:
Consider moving this test to `unittests/others_2`, which has the least CI time.

Reply:
Done